### ISSUE TITLE
Import not pulling in gem_id for Demonology Warlocks

### ISSUE THREAD

**Issue by bradleyjg** [simulationcraft/simc#2992]
When doing an import for a demonology warlock, the weapon string does not have the gem_id part and so the simulator is treating the artifact weapon as having a 750 ilvl. I tested this with "Tulock" from Tichondrius and "Julianodelan" from malganis to make sure it wasn't something idiosyncratic.

**Comment by navv1234** [comment]
Are you using an old version of Simulationcraft? This has been fixed a while ago, at least Julianodelan who is in demonology specialization shows the correct item level of 878 for both the off and the mainhand.

**Comment by bradleyjg** [comment]
I believe I'm using the latest release version. The report shows this version string:
SimulationCraft 703-03 for World of Warcraft 7.0.3 Live (wow build level 22522, git build edb27f3).

I went to the import->import tab, selected realm Mal'Ganis and entered Character Julianodelan then clicked import. 
The returned profile was:

```
warlock="Julianodelan"
origin="https://us.api.battle.net/wow/character/malganis/Julianodelan/advanced"
thumbnail="http://us.battle.net/static-render/us/malganis/146/133485970-avatar.jpg"
level=110
race=undead
role=spell
position=back
professions=mining=759/herbalism=800
talents=http://us.battle.net/wow/en/tool/talent-calculator#VZ!1220221
artifact=37:0:0:0:0:1170:1:1171:3:1173:3:1174:3:1176:3:1177:3:1178:3:1179:1:1180:1:1181:1:1182:1:1183:1:1185:1:1354:1
spec=demonology

...

head=collar_of_enclosure,id=134424,bonus_id=3412/1512/3336
neck=dreamwalkers_pendant,id=139063,bonus_id=3432/1502/3336,enchant=mark_of_the_hidden_satyr
shoulders=amice_of_the_unfurling_tempest,id=134432,bonus_id=1727/1492/1813
back=seacursed_wrap,id=133771,bonus_id=3414/1808/42/1507/1813,gems=150haste,enchant=150int
chest=dreamscale_inlaid_vestments,id=138215,bonus_id=1805/1487
wrists=sindorei_spite,id=132379,bonus_id=1811
hands=handwraps_of_delusional_power,id=138212,bonus_id=1805/1502/3337
waist=roggthread_cord,id=134171,bonus_id=1727/1808/1512/3336,gems=150haste
legs=ragged_horrorweave_leggings,id=139190,bonus_id=1805/1487
feet=cozy_dryad_hoofsocks,id=139194,bonus_id=1807/1487/3337
finger1=loop_of_vitriolic_intent,id=134530,bonus_id=3413/1502/1813,enchant=200haste
finger2=seal_of_saltheril,id=137532,bonus_id=3411/1507/3336,enchant=200haste
trinket1=moonlit_prism,id=137541,bonus_id=1727/1808/1502/3336,gems=150haste
trinket2=chrono_shard,id=137419,bonus_id=1727/1492/1813
main_hand=spine_of_thalkiel,id=137246
off_hand=skull_of_the_manari,id=128943,bonus_id=748,relic_id=3413:1502:1813/1807:1477:3336/1727:1492:1813/0

# Gear Summary
# gear_ilvl=860.38
# gear_stamina=21560
# gear_intellect=13488
# gear_spell_power=9645
# gear_crit_rating=3136
# gear_haste_rating=11343
# gear_mastery_rating=3717
# gear_versatility_rating=279
# gear_speed_rating=320
# gear_armor=1703
# set_bonus=tier19p_cloth_2pc=1
default_pet=felguard

```

I didn't change any of the settings and clicked simulate.

The simulation results were  230191 dps and under the gear section it said:
Average Item Level: 844.00
Local   Main Hand   Spine of Thal'kiel
ilevel: 750, weapon: { 648 - 1206, 2.6 }, stats: { +287 Int, +431 Sta, +265 Haste, +117 Mastery }

Local   Off Hand    Skull of the Man'ari
ilevel: 750, stats: { +219 Int, +328 Sta, +146 Crit, +146 Mastery, +2924 SP }

**Comment by navv1234** [comment]
Yeah, I'm working on it. Will be fixed for the next version.

**Comment by navv1234** [comment]
Fixed in 0393196.
